eslint.config.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const neo = require('neostandard')
+
+module.exports = [
+  ...neo({
+    env: ['jest'],
+    ignores: [
+      'lib/llhttp',
+      'test/fixtures/wpt',
+      'node_modules',
+      'undici-fetch.js'
+    ],
+    ts: true
+  }),
+  {
+    rules: {
+      '@stylistic/comma-dangle': ['error', {
```

Comment:
Would love to get feedback on https://github.com/neostandard/neostandard/issues/79 to make this unnecessary 🙏

Reply:
If you want, you can update to [`0.11.3`](https://github.com/neostandard/neostandard/releases/tag/v0.11.3) now and this override will no longer be needed (though having it will enforce no dangling commas if that's desireable)